src/twisted/conch/test/test_cftp.py

```diff
@@ -59,16 +60,38 @@ class SSHSessionTests(TestCase):
     Tests for L{twisted.conch.scripts.cftp.SSHSession}.
     """
 
-    def test_eofReceived(self):
+    def setUp(self) -> None:
+        self.stdio = FakeStdio()
+        self.channel = SSHSession()
+        self.channel.stdio = self.stdio
+        self.stderrBuffer = BytesIO()
+        self.stderr = TextIOWrapper(self.stderrBuffer)
+        self.channel.stderr = self.stderr
+
+    def test_eofReceived(self) -> None:
         """
         L{twisted.conch.scripts.cftp.SSHSession.eofReceived} loses the write
         half of its stdio connection.
         """
-        stdio = FakeStdio()
-        channel = SSHSession()
-        channel.stdio = stdio
-        channel.eofReceived()
-        self.assertTrue(stdio.writeConnLost)
+        self.channel.eofReceived()
+        self.assertTrue(self.stdio.writeConnLost)
+
+    def test_extReceivedStderr(self) -> None:
```

Comment:
Would it make sense to add a test with invalid UTF-8 data?

Reply:
This test covers that with the `\xff` byte it passes.